src/main/java/Control/ConfigClass.java

```diff
@@ -9,7 +9,7 @@
     // changes db table names from lowercase to camelcase if true and some other things
     public final static boolean RUNNING_ON_SERVER = false;
     // in TimakCommonFiles there is a setup File for creation database, the number should match
-    public final static int CODE_SUPPOSED_TO_WORK_WITH_SQL_VERSION = 13;
+    public final static int CODE_SUPPOSED_TO_WORK_WITH_SQL_VERSION = 14;
```

Comment:
dobry nazov :D "supposed to work", odkial sa tu vzala verzia 14? cislujes ako Microsoft?

Reply:
No odkiaľ sa asi vzala :D Pozri si databázu tabuľku version s riadok version :D v TimakCommonFiles

Reply:
Ja som nepochopila ze to je verzia SQLka, ja ze to je verzia nejakeho kodu a mi to nedavalo zmysel :D ze aky kod, dala by som tam CODE_IS_SUPPOSED_TO_WORK, nech to je uz cela veta a nie pridavne meno, ze kod ktory ma pracovat s SQL verziou, ale to je jedno, len ma to domylilo